diff -ur nvmixer/MixerRegistry/Makefile nvmixer.new/MixerRegistry/Makefile
--- nvmixer/MixerRegistry/Makefile	2005-03-05 04:06:28.000000000 +0000
+++ nvmixer.new/MixerRegistry/Makefile	2005-07-29 16:04:58.629231216 +0000
@@ -8,7 +8,7 @@
 
 INCPATH = -I../../main -I.
 CFLAGS  = -g -Wall $(INCPATH)
-LIBS    = -L../commonlib -lcommonlib
+LIBS    = -L../commonlib -lcommonlib -lm
 
 OBJS = \
      nvmixer-registry.o \
diff -ur nvmixer/commonlib/Makefile nvmixer.new/commonlib/Makefile
--- nvmixer/commonlib/Makefile	2005-03-05 04:06:28.000000000 +0000
+++ nvmixer.new/commonlib/Makefile	2005-07-29 16:04:33.737015408 +0000
@@ -10,12 +10,6 @@
 
 CFLAGS = -g -O2 -Wall -I../.. -I../../main -I.
 
-ifeq (x86_64,$(findstring x86_64,$(shell $(CC) -dumpmachine)))
-LIBS = --start-group -L/usr/lib64 -lm --end-group
-else
-LIBS = --start-group -L/usr/lib -lm --end-group
-endif
-
 OBJS = \
      main.o \
      $(NULL)
@@ -28,7 +22,7 @@
 	$(CC) $(CFLAGS) -M *.c > .depend
 
 $(LIBMAIN): $(OBJS)
-	$(LD) -r $^ -o $@ $(LIBS)
+	$(LD) -r $^ -o $@
 
 $(TARGET): $(LIBMAIN)
 	$(AR) r $@  $(LIBMAIN)
